recipes/python/regex.yaml

```diff
@@ -0,0 +1,18 @@
+inherit: ['python3::setuptools', 'python3::cext']
+
+metaEnvironment:
+    PKG_VERSION: "2025.2.13"
+    PKG_DESCRIPTION: "Python3 regex packet (extends re)"
+    PKG_LICENSE: "Apache-2.0"
+
+checkoutSCM:
+    scm: git
```

Comment:
Sorry, but could we use a tar archive instead?

Reply:
There are no releases on GitHub? Do you know other sources?

Reply:
Github also provides tarballs for tags: https://github.com/mrabarnett/mrab-regex/releases/tag/2025.2.13

Reply:
done